MaraBot/Program.cs

```diff
@@ -41,9 +48,10 @@ static async Task MainAsync()
             commands.RegisterCommands<Commands.PresetsCommandModule>();
             commands.RegisterCommands<Commands.PresetCommandModule>();
             commands.RegisterCommands<Commands.RaceCommandModule>();
+            commands.RegisterCommands<Commands.CreatePresetCommandModule>();
```

Comment:
I'm a bit uneasy about this command, but I understand what you're trying to do. I'm ok with the bot being able to nicely format presets as long as we don't add them to the preset list.

That said, it might be a bit spammy on the channel. Writing your preset through a bot might not be the best way to do it.

Reply:
I have made it DM-specific to remove any spam potential.